### ISSUE TITLE
Volume optimization of implant

### ISSUE THREAD

**Issue by a3sroot** [BishopFox/sliver#562]
1、At present, the volume of binary files generated is relatively large. If you continue to add modules later, sliver may be more like a tool set than a C2. Recently, I have also been studying some hot update schemes to reduce the volume of binary. Although it can also be realized by using loader, the volume problem still needs to be solved after all.  😔
      
      2、On the server, you can edit some yaml to realize the contents of some planned tasks, such as regular execution, online execution and execution under certain conditions.  😋

tip:
     Characteristics of golang TLS ja3 fingerprint -> https://github.com/CUCyber/ja3transport

**Comment by a3sroot** [comment]
In the current form of Internet attack, it is better to directly replace HTTP with WS, which is nothing more than adding a layer of TLS.

**Comment by moloch--** [comment]
Yes this will be a priority for us in v1.6, however I'd point out the best approach is to already use a stager, which can be as small as a few hundred bytes.
